Makefile

```diff
@@ -7,7 +7,7 @@ install: ## Install the dependencies
 
 .PHONY: develop
 develop: ## Install the test and dev dependencies
-	python3 -m pip install -e .[test,dev]
+	python3 -m pip install -e .[test,dev,sync]
```

Comment:
Seeing this makes me also think we should add this group in the tests, when running the CI (not sure if it uses `make develop`).

Reply:
It does use `make develop`, I've checked :)